src/ResourceManager/IotHub/Commands.IotHub.Test/ScenarioTestFile/PS-Test-Root-CA.cer

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Whats the expiration for this cert? Will test start failing after a certain date?

Reply:
Updated new test certificate with 10 yrs of expiry